GOVERNANCE.md

```diff
@@ -0,0 +1,118 @@
+# Operator Framework Charter
+
+v1.0
+
+2019-10-16 
+
+## Introduction
+The charter describes the logistics of the Operator Framework community project.
+
+The Operator Framework is an open-source project based on the Apache Licence that provides tooling for software engineers and Kubernetes cluster owners to develop and run Kubernetes Operators at scale.
```

Comment:
```suggestion
The Operator Framework is an open-source project based on the Apache Licence 2.0 that provides tooling for software engineers and Kubernetes cluster owners to develop and run Kubernetes Operators at scale.
```

Reply:
WDYT?

Reply:
"tooling and runtimes" if this becomes a multi-faceted project.